### ISSUE TITLE
Is --enable-period-mode still supported >=6.3.0?

### ISSUE THREAD

**Issue by dyronald** [dashingsoft/pyarmor#429]
It seems like --enable-period-mode no longer has effect when running version 6.3.0. Replacing the _license.lic_ file only takes effect when the app is restarted.

For comparison, I also ran the code below in a loop while replacing the _license.lic_ file in the background. 
```
pytransform.pyarmor_init(is_runtime=1)
days_left = pytransform.get_expired_days()
```
In 6.2.8, **days_left** is updated when the _license.lic_ file is replaced.
In 6.3.0 and 6.5, **days_left** is only updated after the app is restarted.

**Comment by jondy** [comment]
From 6.3.0, by default the license is embedded into the binary files. Please obfuscate the scripts with outer license 
https://pyarmor.readthedocs.io/en/latest/advanced.html#how-to-use-outer-license-file

Alse refer to
https://pyarmor.readthedocs.io/en/latest/change-logs.html#incompatible-issues

**Comment by dyronald** [comment]
Thanks for the quick response.

I did use outer for 6.3.0. Here's the command I used:
`pyarmor obfuscate --with-license=outer --platform=linux.x86_64 --recursive --output=dist/app app/__init__.py`

And for 6.2.8:
`pyarmor obfuscate --platform=linux.x86_64 --recursive --output=dist/app app/__init__.py`

pyarmor 6.3.0 does pick up the outer license. The issue is that it doesn't detect an updated _license.lic_ while app is running. The app needs to restarted for the new _license.lic_ to be used. When running on 6.2.8, the app does not need restarting.

**Comment by jondy** [comment]
The check interval is about 3600 secs (1 hour), are you sure the new license doesn't work after 1 hour?

**Comment by dyronald** [comment]
In my test, I was replacing a valid license with an expired one. 
On 6.3.0, the app keeps working even more than an hour after replacing. But would correctly detect the expired license (and not work) if I restart it.

**Comment by jondy** [comment]
I check the source, the latest version should check the license per hour as v6.2.8 does, but the checkpoint is executed only when any obfuscated function is called. 

Do you test same app in 6.2.8 and 6.3.0?

**Comment by dyronald** [comment]
Yes. Exactly the same app. Only pyarmor version was changed. I'll try the latest version.

**Comment by jondy** [comment]
I find the problem, I'll build a pre-release version 6.6.1 for linux.x86_64.

**Comment by dyronald** [comment]
Great! Thanks for the quick response.

**Comment by jondy** [comment]
Please download pre-release v6.6.1 or install it by pip directly 
https://pyarmor.dashingsoft.com/downloads/temp/pyarmor-6.6.1.zip

**Comment by jondy** [comment]
Fixed in v6.6.1
